apps/api/src/deployment/services/deployment/deployment.service.ts

```diff
@@ -193,4 +194,33 @@ export class DeploymentService {
       });
     }
   }
+
+  public async list(
+    owner: string,
+    { skip, limit }: { skip?: number; limit?: number }
```

Comment:
if `limit` is enforced it would be a required param here with no need to check below

Reply:
I'll keep both optional but with a default of 1000 for the limit, makes sense?